src/model/Ticket.java

```diff
@@ -30,31 +31,28 @@ public static Ticket getInstance() {
 		return instance;
 	}
 	
-	public static void addTicket(ArrayList<Product> productos) throws IOException {
```

Comment:
Toda esta parte de ticket queda comentada. Todo lo que hay en el for deberá pasar al controlador, el delete ahora espera un objeto y si hacemos que ticket notifique a stock aún habrá que cambiar esta clase aún más.

Reply:
Vale, pero ahora mismo no está en el controlador. Y, como te he dicho, habría que dejar solo un Controller por cada Model